packages/atomic-hosted-page/src/components/atomic-simple-builder/atomic-simple-builder.tsx

```diff
@@ -0,0 +1,73 @@
+import {Schema, StringValue} from '@coveo/bueno';
+import {Component, ComponentInterface, Method, Element} from '@stencil/core';
+import {processHostedPage} from '../atomic-hosted-page/hosted-pages';
+
+interface AtomicSimpleBuilderInitializationOptions {
+  /**
+   * The unique identifier of the search interface.
+   */
+  interfaceId: string;
+  /**
+   * The unique identifier of the target Coveo Cloud organization (e.g., `mycoveocloudorganizationg8tp8wu3`)
+   */
+  organizationId: string;
+  /**
+   * The access token to use to authenticate requests against the Coveo Cloud endpoints. Typically, this will be an API key or search token that grants the privileges to execute queries and push usage analytics data in the target Coveo Cloud organization.
+   */
+  accessToken: string;
+  /**
+   * The Plaform URL to use. (e.g., https://platform.cloud.coveo.com)
+   * The platformUrl() helper method can be useful to know what url is available.
+   * @defaultValue `https://platform.cloud.coveo.com`
+   */
+  platformUrl?: string;
+}
+
+/**
+ * A Web Component used to inject a [Coveo Search Interface made with the simple builder](https://docs.coveo.com/en/m7e92019/adobe/build-the-search-solution-using-a-coveo-ui-library-directly#search-interface-builder) in the DOM.
+ * Pulls from the [Search Interfaces API](https://platform.cloud.coveo.com/docs?urls.primaryName=Search%20Interface%20Service#/)
+ * @internal
+ */
+@Component({
+  tag: 'atomic-simple-builder',
```

Comment:
`atomic-simple-builder` makes me think that we build the page in Atomic rather than it being hosted somewhere else.

I would prefer a name like `atomic-hosted-simple-page` for this component, and the other component could keep its current name.